src/iso_alloc.c

```diff
@@ -76,7 +76,9 @@ INTERNAL_HIDDEN void verify_zone(iso_alloc_zone_t *zone) {
 }
 
 INTERNAL_HIDDEN void _verify_all_zones(void) {
-    for(int32_t i = 0; i < _root->zones_used; i++) {
+    size_t zones_used = _root->zones_used;
```

Comment:
Maybe mark it `const`, to make this explicit?

Reply:
Yes, they should be `const`. Im implicitly casting to `size_t` here because it will be converted in the for loop comparison anyway.